local_postgres_docker_setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Is this for a production situation? I thought this was mostly for dev, with RDS or equivalent used n production.